docs/gnode.md

```diff
@@ -0,0 +1,12 @@
+---
+state: design # Nothing implemented yet, figuring out the best way to do it.
+---
+
+# Gnodes
+
+Gnodes are interconnected [DAOs](./daos.md) for scaling and specialized communities. They enable secure collaboration and seamless integration, facilitating transparent ecosystem management at scale.
+
+Layers and overlays for different perspectives:
+- first layer: entities, people, and connections to projects and teams.
```

Comment:
Not sure if the punctuation at the end is a mistake, since other list items don't have it